Editor/AGS.Editor/Panes/FreePanControl.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Text;
+using System.Windows.Forms;
+
+namespace AGS.Editor
+{
+    public class FreePanControl : BufferedPanel
```

Comment:
Your ordering in the class seems to be random. Please organize your class so that things appear in the following order

1. Fields like `private int field;`
2. Properties `public int Field { get; }`
3. Constructor `public FreePanControl() {}`
4. Functions `public void TheFunction() {}`

Reply:
I pushed a new commit below, please check if it fixes the ordering. 

Also, some winforms method appear to not have explicitly mentioned public/private in the docs. Should I add the public keywords in these?
https://docs.microsoft.com/en-us/dotnet/api/system.windows.forms.control.createparams?view=netframework-4.5

Reply:
The one you linked to in the doc is `protected` instead of private/public. It means private, but also accessible for classes that inherits from the class that declares the function.

Reply:
Anyway the ordering looks good now. I haven't checked anything else in this PR

Reply:
Thanks, I will click this resolve conversation button and squash these two commits into one.